.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -0,0 +1,36 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: '[Feat]: '
+labels: ['feature', 'triage']
+assignees: ''
+
+---
+
+**Describe the solution you'd like:**
+
+[A clear and concise description of what you want to happen.]
+
+**User value:**
+
+[Why will this feature be valuable to you? Why will this be valuable to others?]
+
+**Expected behavior:**
+
+[What would you like to see happen]
+
+**Proposed solution:**
+
+[If you're able, describe possible solution workflow]
+
+**Anything else you would like to add:**
+
+[Miscellaneous information that will assist in solving the issue.]
+
+**Testing changes required:**
+
+[List possible testing changes required, if none please explain]
+
+**Documentation changes required:**
```

Comment:
Just an idea, but maybe it would be a nice idea to add that if the user is not sure about the documentation and testing changes required, they can mark it as so. It would be nice though to think of a flow where not exhaustively declaring where the documentation needs updated causes a roadblock in the feature's development.